JitsiMeetJS.ts

```diff
@@ -194,6 +197,14 @@ export default {
         Logger.setLogLevel(level);
     },
 
+    getRtcStatsTrace() {
```

Comment:
Can we group the API in an object so we can use it like so: `JitsiMeetJS.rtcstats.getTrace()` ?